### DISCUSSION TITLE
https://ericswpark.com/blog/2023/2023-06-17-migrating-homebrew-from-intel-to-asi-mac/

### DISCUSSION THREAD

**Discussion by giscus** [ericswpark/ericswpark.github.io-comments#12] [Announcements]
# https://ericswpark.com/blog/2023/2023-06-17-migrating-homebrew-from-intel-to-asi-mac/

Student Developer

https://ericswpark.com/blog/2023/2023-06-17-migrating-homebrew-from-intel-to-asi-mac/

**Comment by porg** [comment]
> Go into /usr/local and delete all files and folders in there after it had finished running. It should usually be safe to delete everything in that folder since macOS doesn’t even ship with it by default

- That information is very wrong!
- A lot of other tools and also package managers (npm, python, make builds, etc) install into `/usr/local`.
- So you should carefully remove Homebrew related files/directories only, else you may ruin the other setups.

Related discussion:
https://apple.stackexchange.com/questions/437618/why-is-homebrew-installed-in-opt-homebrew-on-apple-silicon-macs

**Reply by ericswpark** [reply]
Yup, thank you for the heads up. I do mention on the next sentence to delete it carefully in case:
> you do have things that depend on stuff inside that folder, I suggest deleting carefully so that you don’t end up breaking something.

I will add the disclaimer about other tools and package managers shortly.
